pkg/specgen/generate/config_linux.go

```diff
@@ -79,7 +79,7 @@ func addPrivilegedDevices(g *generate.Generator) error {
 }
 
 // DevicesFromPath computes a list of devices
-func DevicesFromPath(g *generate.Generator, devicePath string) error {
+func DevicesFromPath(g *generate.Generator, devicePath string) (string, error) {
```

Comment:
You add a new return value to this function but no caller uses it. Please remove it if it is not needed.